### ISSUE TITLE
Layer normalization

### ISSUE THREAD

**Issue by goonm** [Eydcao/BSMS-GNN#1]
Thank you for sharing your good research findings. 
I have a question about layer normalization in your network. 
Unlike conventional cnn or transformer, one layer normalization is applied after going through several gcn. 
Is there a special reason for applying it at the end of the encoder instead of applying it after each gcn layer?

**Comment by Eydcao** [comment]
Dear Goonm, sorry for the delay.

I seldom check the status on GitHub. Concerning your question, I am not quite sure if I get this correctly: the layer normalization is applied at the end of each MLP. This is defined in `ops.py` and

```Python
class MLP(torch.nn.Module):
    def __init__(self, input_dim, latent_dim, output_dim, hidden_layers, layer_normalized=True):
...
```

I believe **the only place we do not apply a normalization is the decoder.**

Let me know your thoughts,
Yadi
